emcc.py

```diff
@@ -2367,8 +2382,11 @@ def get_full_import_name(name):
   if settings.WASMFS:
     settings.LINK_AS_CXX = True
 
-  if settings.RELOCATABLE and settings.EXCEPTION_HANDLING:
-    settings.DEFAULT_LIBRARY_FUNCS_TO_INCLUDE.append('__cpp_exception')
+  if settings.RELOCATABLE:
```

Comment:
Perhaps a comment here.  Something like `export Tag objects which are likely needed by the native code, but which are currently not reported in the metadata of wasm-emscripten-finalize`?

Reply:
Done